editable-samples-2/pages/filter/index.html

```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width" />
-    <link href="../../css/editable.css" rel="stylesheet"></link>
+    <link href="../../css/editable.css" rel="stylesheet" />
     <link href="../../css/prism.css" rel="stylesheet">
```

Comment:
We should unify both endings here.
```suggestion
    <link href="../../css/editable.css" rel="stylesheet" />
    <link href="../../css/prism.css" rel="stylesheet" />
```